weavertest/init.go

```diff
@@ -131,11 +137,11 @@ func Run(t testing.TB, mode Mode, opts Options, testBody any) {
 		}
 	}()
 
-	if mode == Local {
-		ctx = initSingleProcess(ctx, opts.Config)
+	if !r.multi && !r.forceRPC {
+		ctx = initSingleProcess(ctx, r.config)
```

Comment:
nit: this is initSingleProcessLocal i think

Reply:
Done.